service/src/main/java/bio/terra/pipelines/app/controller/GlobalExceptionHandler.java

```diff
@@ -104,7 +104,7 @@ public ResponseEntity<ApiErrorReport> retryBackoffExceptionHandler(
   // -- catchall - log so we can understand what we have missed in the handlers above
   @ExceptionHandler(Exception.class)
   public ResponseEntity<ApiErrorReport> catchallHandler(Exception ex) {
-    logger.warn("Exception caught by catchall handler", ex);
+    logger.warn("Exception caught by catchall handler: {}", ex.getMessage());
```

Comment:
the `buildApiErrorReport` function already logs the full exception so i figured instead of doubling up in the logs we could instead just log the message here and connect that with the `buildApiErrorReport` log if necessary